dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestEDR.war/resources/index.jsp

```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2021 IBM Corporation and others.
+    Copyright (c) 2022 IBM Corporation and others.
```

Comment:
Hey Evan, the date should be a range now. The file was initially checked into git in 2021 and then updated in 2022 so it should be 2021, 2022

Reply:
done